plugins/Flanger/FlangerControlsDialog.cpp

```diff
@@ -38,24 +38,24 @@ FlangerControlsDialog::FlangerControlsDialog( FlangerControls *controls ) :
 	QPalette pal;
 	pal.setBrush( backgroundRole(), PLUGIN_NAME::getIconPixmap( "artwork" ) );
 	setPalette( pal );
-	setFixedSize( 200, 75 );
+	setFixedSize( 195, 75 );
 
 	Knob* delayKnob = new Knob( knobBright_26, this );
-	delayKnob->move( 20,10 );
+	delayKnob->move( 10,10 );
 	delayKnob->setVolumeKnob( false );
 	delayKnob->setModel( &controls->m_delayTimeModel );
 	delayKnob->setLabel( tr( "Delay" ) );
 	delayKnob->setHintText( tr( "Delay Time:" ) + " ", "s" );
 
 	TempoSyncKnob * lfoFreqKnob = new TempoSyncKnob( knobBright_26, this );
-	lfoFreqKnob->move( 53,10 );
+	lfoFreqKnob->move( 45,10 );
 	lfoFreqKnob->setVolumeKnob( false );
 	lfoFreqKnob->setModel( &controls->m_lfoFrequencyModel );
 	lfoFreqKnob->setLabel( tr( "Lfo Hz" ) );
 	lfoFreqKnob->setHintText( tr ( "Lfo:" ) , "Hz" );
 
 	Knob * lfoAmtKnob = new Knob( knobBright_26, this );
-	lfoAmtKnob->move( 86,10 );
+	lfoAmtKnob->move( 84,10 );
 	lfoAmtKnob->setVolumeKnob( false );
 	lfoAmtKnob->setModel( &controls->m_lfoAmountModel );
 	lfoAmtKnob->setLabel( tr( "Amt" ) );
```

Comment:
I'd rename `Amt` to `Amnt`, as that's what its called in the Peak Controller and LFO controller. Furthermore, I believe we should convert the labels to all caps, to be consistent with the other plugins: 
![screenshot from 2016-08-29 01 48 58](https://cloud.githubusercontent.com/assets/6282045/18037701/d8fc8b8c-6d8a-11e6-8dec-a8a659838851.png)